### DISCUSSION TITLE
Allure report at build level (only executed tests)

### DISCUSSION THREAD

**Discussion by ghost** [allure-framework/allure2#2724] [Questions & Support]
Hi,

Is it possible to display test result at build level (only tests that have been executed during the build)  because on generated report, we find all tests present in previous history and can't select only executed tests in the last build

Thanks for your feddback

**Comment by baev** [comment]
You need to clean allure-results directly before the build

**Comment by ghost** [comment]
Yes I do it

${OUTPUT_WORKSPACE}/allure contains results generated by Robot Framework during build execution

                                        rm -Rf $WORKSPACE/allure-results
                                        mv ${OUTPUT_WORKSPACE}/allure  $WORKSPACE/allure-results/
                                        if [ -d "$WORKSPACE/allure-report/history" ];then
                                                cp -r $WORKSPACE/allure-report/history $WORKSPACE/allure-results/history;
                                        fi

In after in the Jenkins pipeline :
                        post {
                always {
                    allure includeProperties:
                     false,
                     jdk: '',
                     properties: [],
                     reportBuildPolicy: 'ALWAYS',
                     results: [[path: '${WORKSPACE}/allure-results']]
                }

Vincent
